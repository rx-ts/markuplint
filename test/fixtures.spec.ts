import path from 'path'

import { exec } from 'markuplint-sync'

describe('fixtures', () => {
  it('should just work', () => {
    const resultInfos = exec({
      files: path.resolve(__dirname, 'fixtures/*.html'),
    })

    for (const resultInfo of resultInfos) {
      expect(resultInfo.results).toMatchSnapshot(
        path.relative(__dirname, resultInfo.filePath),
      )
    }
  })
})
