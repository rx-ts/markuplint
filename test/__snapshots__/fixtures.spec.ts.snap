// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`fixtures should just work: fixtures/attr-duplication.html 1`] = `
Array [
  Object {
    "col": 24,
    "line": 1,
    "message": "Duplicate attribute name",
    "raw": "data-Attr",
    "ruleId": "attr-duplication",
    "severity": "error",
  },
]
`;
